qutebrowser/browser/shared.py

```diff
@@ -79,14 +79,23 @@ def authentication_required(url, authenticator, abort_on):
     return answer
 
 
+def _format_msg(msg: str) -> str:
+    """Convert message to HTML suitable for rendering."""
+    ret = msg
```

Comment:
Why not use `msg` directly below?

Reply:
a matter of style i guess, so the lines that make changes can be rearranged freely. but i'm happy to change it